docker/files/run/run.sh

```diff
@@ -59,7 +64,12 @@ function configure_data_dir() {
 
   # check for token overrides
   if [ "$TOKEN" != "" ]; then
+    # set token for relay node
     echo "$TOKEN" >algod.token
+    # set token for follower node
+    if [ -d "${ALGORAND_DATA}/../follower/" ]; then
+       echo "$TOKEN" >"${ALGORAND_DATA}/../follower/algod.token"
+    fi
```

Comment:
Is there a more general way to do this? What if the template has 3 nodes and different names?

Reply:
updated.